applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -1876,13 +1876,11 @@ void UPwSmallStrainInterfaceElement<TDim, TNumNodes>::CalculateAndAddCouplingMat
 {
     KRATOS_TRY
 
-    noalias(rVariables.UDimMatrix) = prod(trans(rVariables.Nu), trans(rVariables.RotationMatrix));
-
-    noalias(rVariables.UVector) = prod(rVariables.UDimMatrix, rVariables.VoigtVector);
+    Matrix b_matrix = prod(rVariables.RotationMatrix, rVariables.Nu);
```

Comment:
The stress state policy also has a function to calculate the b matrix, does that not work for interface elements?

Reply:
Sorry for confusion, it is not really a B matrix. Changed the name as like_b_matrix